src/components/progress-bar/progress-bar-styles.jsx

```diff
@@ -84,6 +88,12 @@ export default theme => {
 
     progressBar: {
       display: 'flex',
+      flexWrap: 'wrap',
+
+      '&.xs': { margin: -sizeDict.xs.radius / 2 },
+      '&.sm': { margin: -sizeDict.sm.radius / 2 },
+      '&.md': { margin: -sizeDict.md.radius / 2 },
+      '&.lg': { margin: -sizeDict.lg.radius / 2 },
```

Comment:
You could replace the four lines above with `margin: ({ size }) => -sizeDict[size].radius / 2,`. What do you think @iamstarkov?